src/sentry/static/sentry/app/views/eventsV2/table/tableModalEditColumn.tsx

```diff
@@ -292,30 +289,13 @@ const FormRow = styled('div')`
   grid-column-gap: ${space(2)};
 `;
 
-const FooterContent = styled('a')`
+const FooterContent = styled(Link)`
   display: flex;
+  align-items: center;
+  flex-grow: 1;
```

Comment:
The `flex-grow` implementation results in a smaller area for clicking into the link.

![image](https://user-images.githubusercontent.com/1748388/74294059-98a4fc00-4cf1-11ea-9a6f-048a0f629ee2.png)


With the hack, the entire footer is used, which IMO is slightly better UX:

![image](https://user-images.githubusercontent.com/1748388/74294035-8d51d080-4cf1-11ea-8515-f4cfe571d65b.png)


While I prefer better UX, I'm not really opinionated on this specific instance. If you are aware of the change and want to go ahead with it, I'm happy to approve.

Reply:
I'm not too concerned about this - chances are they'll just hit the words "Read the docs"